backend/de.metas.fresh/de.metas.fresh.base/src/main/sql/postgresql/system/70-de.metas.fresh/5698120_sys_gh16021_interim_flatrate_conditions_with_modular_contract_settings.sql

```diff
@@ -0,0 +1,54 @@
+-- Column: C_Flatrate_Conditions.C_Interim_Invoice_Settings_ID
```

Comment:
Please, move this file in de.metas.contracts where the other scripts related to modular contracts were added